include/opendht/indexation/pht.h

```diff
@@ -166,6 +175,125 @@ class Pht {
     void insert(Key k, Value v, Dht::DoneCallbackSimple cb = {});
 
 private:
+    class Cache {
+    public:
+        /**
+         * Insert all needed node into the tree according to a prefix
+         * %param p : Prefix that we need to insert
+         */
```

Comment:
This is a minor observation: the format of documentation should be consistent with the rest of the opendht code, i.e. with [doxygen format][]. I think `%param` is not part of the standard format. Use `@param` instead. 

[doxygen format]: https://www.stack.nl/~dimitri/doxygen/manual/docblocks.html